pkg/services/ngalert/api/tooling/definitions/convert_prometheus_api.go

```diff
@@ -82,6 +82,38 @@ import (
 //       403: ForbiddenError
 //       404: NotFound
 
+// Route for mimirtool / UI
```

Comment:
Nit: not sure that it's "for mimirtool" and "for cortextool" since they don't support these endpoints :)